HashTable.java

```diff
@@ -0,0 +1,107 @@
+class HashTable{
```

Comment:
Time & Space?

Reply:
Time in case of insert should be O(1) but in case of search worst case will be still O(n) but I feel it will perform better as we are utilizing hashfunctions even to search the index and then to check whether the element is present or not.
Space complexity would be O(N) as we are using one String array to store the element.,